jmix-audit/audit-flowui/src/main/java/io/jmix/auditflowui/view/entitylog/EntityLogView.java

```diff
@@ -300,6 +301,19 @@ protected void onInit(View.InitEvent event) {
         setupWrapper.setColspan(loggedEntityTableBox, 2);
     }
 
+    protected Stream<String> onUserFieldFetchCallback(Query<String, String> query) {
+        query.getOffset();
```

Comment:
Is this extra code?

Reply:
what do you mean by "extra"?

Reply:
query.getOffset(); is required to be called  exception will be thrown in runtime otherwise

Reply:
Add a comment, please. For example:
_Ignore offset and limit since methods called to avoid exception_